exasol_transformers_extension/resources/templates/model_loader_udf.jinja.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE {{ language_alias }} SET SCRIPT "TE_MODEL_LOADER_UDF"(
```

Comment:
This makes no sense to have a UDF in the package, that only loads the model

Reply:
Please see updated PR, removing this UDF from the set of common UDFs and only instantiating and using it in the integration tests now.